test_aodndata/moorings/test_dwmHandler.py

```diff
@@ -13,7 +13,9 @@
 IMAGES_ZIP = os.path.join(TEST_ROOT, IMAGES_ZIP_BASENAME)
 IMAGES_ZIP_CONTENTS = {'saz_2012_47_1000_01.jpg', 'saz_2012_47_1000_02.JPG', 'saz_2012_47_1050_IRS_01.tiff'}
 MIXED_ZIP = os.path.join(TEST_ROOT, 'images_and_nc.zip')
-
+CALIBRATION_ZIP_BASENAME = 'calibration_Pulse-6-2009.zip'
```

Comment:
We don't really need a 22Mb file for the purpose of this test. It would be nice (non necessary) to make these test files smaller. They can even contain just empty files, since the only thing that matters is their extension.

Reply:
I was using a 'reduced' version of the file before, but when all the tests were running fine I decided to change to the original file. But I agree, I made the test file smaller now.